theme/management/commands/add_quota_avu.py

```diff
@@ -30,6 +34,16 @@ def handle(self, *args, **options):
                             print res.short_id + ' does not have an owner'
                             continue
                     res.set_quota_holder(res.creator, res.creator)
+
+                # Now reset the quota system by setting the AVU 'resetQuotaDir' to anything
+                istorage = res.get_irods_storage()
+                # Add a specific AVU to invoke quota recalculation
+                istorage.session.run("imeta", None, 'set', '-C', res.root_path,
+                                     'resetQuotaDir', '0')
+                # Now remove that AVU to clean up; this is not part of the resource API
+                istorage.session.run("imeta", None, 'rm', '-C', res.root_path,
+                                     'resetQuotaDir', '0')

```

Comment:
@phuongdm OK. Got it. Will look at your microservice codes when I have a chance.